### PR TITLE
Show alert for major outages

### LINKED ISSUE #1342: Status summary bar at top doesn't show red for major outage

### PR DESCRIPTION
**Pull request by jbrooksuk**
Closes #1342 

We need to figure out how this is going to work, a major service wouldn't want to show a major outage warning if there are loads of services, whereas a small one would.

Ping @CachetHQ/owners

### CODE REVIEW

File: resources/lang/en/cachet.php [hunk 1]
```diff
@@ -40,8 +40,9 @@
 
     // Service Status
     'service' => [
-        'good' => 'All systems are functional',
-        'bad'  => 'Some systems are experiencing issues',
+        'good'  => 'All systems are functional',
+        'bad'   => 'Some systems are experiencing issues',
+        'major' => 'One or more systems are experiencing a major outage',
```
Comment (GrahamCampbell) [inline resources/lang/en/cachet.php]: What if a system only has one compnent?

File: resources/lang/en/cachet.php [hunk 2]
```diff
@@ -40,8 +40,9 @@
 
     // Service Status
     'service' => [
-        'good' => 'All systems are functional',
-        'bad'  => 'Some systems are experiencing issues',
+        'good'  => 'All systems are functional',
+        'bad'   => 'Some systems are experiencing issues',
```
Comment (GrahamCampbell) [inline resources/lang/en/cachet.php]: Same here I guess, lol.

File: resources/lang/en/cachet.php (hunk 1 above)
Comment (jbrooksuk) [inline resources/lang/en/cachet.php]: It's only temporary :)

File: resources/lang/en/cachet.php (hunk 2 above)
Comment (jbrooksuk) [inline resources/lang/en/cachet.php]: This translation probably does need changing actually.

**Comment by jbrooksuk** [comment]
Ping @GrahamCampbell :)

**Comment by GrahamCampbell** [comment]
Needs rebasing, language files need syncing, and squashing before merging.

**Comment by jbrooksuk** [comment]
I'll do this tonight so we can get it in.

**Comment by jbrooksuk** [comment]
This change is basically going to break Cachet for all languages (as in, they'll all be in English), so I'll have to update CrowdIn.

**Comment by jbrooksuk** [comment]
Balls. I clicked the "Update branch" button when switching window. URGH!

**Comment by TheoBearman** [comment]
Noob!

**Comment by GrahamCampbell** [comment]
> Balls. I clicked the "Update branch" button when switching window. URGH!

You can still rebase and force push to the branch. ;)

**Comment by jbrooksuk** [comment]
True!

**Comment by jbrooksuk** [comment]
I'm going to sync up all of the language files over the weekend, it's a ball ache.

**Pull request merged**